markdownlint/TOP005_blanksAroundMultilineHtmlTags/TOP005_blanksAroundMultilineHtmlTags.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
      * We only want to flag HTML tags we use for actual markup,
      * or md code block examples of such.
      */
-    const IGNORED_FENCE_TYPES = ["html", "jsx", "erb", "ejs"];
+    const IGNORED_FENCE_TYPES = ["html", "jsx", "erb", "ejs", "ruby"];
```

Comment:
Since you raised the Ruby edge case, even if we're not likely to need it, we *might* have a situation where we have like a template literal with HTML in it in JS. Might as well cover it while we're here.

```suggestion
    const IGNORED_FENCE_TYPES = ["html", "jsx", "erb", "ejs", "ruby", "javascript"];
```